backend/bcssm_backend/routes/users.py

```
@@ -200,6 +200,73 @@ def get_users():
+        })
+
+        try:
+            user_cache_key = f'user:data:{name}'
+            cache.set(user_cache_key, {
+                'id': user_id,
+                'name': name,
+                'role': role,
+                'section_name': section_name,
+                'is_leader': is_leader,
+            }, timeout=1800)
+        except Exception as cache_error:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)